mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -52,33 +52,49 @@ class DecisionTreeClassifier @Since("1.4.0") (
 
   // Override parameter setters from parent trait for Java API compatibility.
 
+  /** @group setParam */
```

Comment:
I am sorry but please allow me leave another trivial comment about documentation. I haven't checked the built documentation but it seems we should use multiple-lines in this case (see https://github.com/apache/spark/pull/13855).